discord/message.go

```diff
@@ -23,6 +23,9 @@ type Message struct {
 	// Flags are the MessageFlags.
 	Flags MessageFlags `json:"flags"`
 
+	// The message associated with the message_reference. This is a minimal subset of fields in a message (e.g. author is excluded.)
+	MessageSnapshots []Message `json:"message_snapshots,omitempty"`
```

Comment:
Is Author always omitted? If it is, please make a copy of the Message type named MessageSnapshot or something with exactly the fields present.

Reply:
Reference: https://discord.com/developers/docs/resources/message#message-snapshot-object

Reply:
Wait, this field is wrong then. It's

```go
MessageSnapshots []struct{ Message *Message } // pseudo-code
```

and not

```go
MessageSnapshots []Message
```